pom-modify/pom-modify.py

```diff
@@ -0,0 +1,113 @@
+import xml.etree.ElementTree as ET
+import re
+import xml.dom.minidom as minidom
+import os
+import sys
+
+def nondex_xml_element():
+    plugin = ET.Element('plugin')
+    
+    groupId = ET.SubElement(plugin, 'groupId')
+    groupId.text = 'edu.illinois'
+    
+    artifactId = ET.SubElement(plugin, 'artifactId')
+    artifactId.text = 'nondex-maven-plugin'
+    
+    version = ET.SubElement(plugin, 'version')
+    version.text = '1.1.2'
+
+    return plugin
+
+def idflakies_element():
```

Comment:
Since this is the project for NonDex, there is no need for the modification script here to add iDFlakies.